### PR TITLE
508-defect-3 [SCREENREADER]: STEM Standalone - Links that open in new tab do not alert user that a new tab will be opened

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
## Description
As a screen reader who cannot see and uses the keyboard to navigate, when tabbing to various links in the form, they do not announce to the user that they open in a new tab. User might think the links will exit the user from the form.
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/18247
## Testing done
Local Test
Mac - Apple Voiceover
Windows - JAWS

## Acceptance criteria
- [x] Add hidden screen reader only text to link text that announces it opening in a new window or give link aria-label that includes text that announces opening in a new window

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by delli-gatti-michael — APPROVED** [review]
lgtm

File: src/platform/forms/preSubmitInfo.js
Comment (bkjohnson) [inline src/platform/forms/preSubmitInfo.js:18]: @1Copenut  or @noahgelman, is there a downside to using `aria-label` to indicate `will open in new tab`?

Comment (1Copenut) [reply src/platform/forms/preSubmitInfo.js:18]: No, that appears to be a reasonable way to announce links are opening in a new tab. The one downside is that Google Translate may not switch `aria-label` to Spanish. We're moving in a direction where there will be more Spanish content, so knowing these accessible labels are there will be important in any future translations.

File: src/applications/edu-benefits/10203/content/benefitSelection.jsx
Comment (1Copenut) [inline src/applications/edu-benefits/10203/content/benefitSelection.jsx:5]: Aria-labels shouldn't be needed on the `<p>` and `<span>` tags. They're intended for adding accessible text or more context for links and buttons, but don't appear to be changing meaning in these text nodes.

**Review by 1Copenut — CHANGES_REQUESTED** [review]
I left a comment about the `aria-labels` not being needed for P and SPAN tags. The updates look good for the links, announcing that they will open in new tabs.

**Review by bkjohnson — APPROVED** [review]
Looks good to merge once Trevor approves.

**Review by 1Copenut — APPROVED** [review]
LGTM!

**Comment by noahgelman** [comment]
Approved

**Pull request merged**
